City Gen version 2

tkwebster opened the pull request:
1. Set the title to exactly what you are adding.

2. Which team is this for?
Gameplay

3. Give a short description of what you are implementing:

- Building Walls no longer overlap
- Invisible Walls added
- Wall colors added as a test feature
- City Parameters are now synced, allowing for the parameters to be chosen by host in future

liaoto committed:
Arena Walls (not invisible)

tkwebster committed:
InvisibleWalls+BuildingColor

JasonStallkamp committed:
Added Network and file synced city generation config

tkwebster committed:
Merge pull request #27 from OSUGameDev/CityGen2Networkingconfig

Added Network and file synced city generation config

tkwebster committed:
Working on Building Generation

tkwebster committed:
Building Overlap Removed

Building will no longer overlap

JasonStallkamp commented on Local War/Assets/Resources/Script/CityGen/MeshGenerator.cs:
```diff
@@ -183,22 +188,87 @@ void BuildWall(float px, float py, float pz, float sx, float sy, float sz)
+		else 
+		{
+			cubeRenderer.material.SetColor("_Color", teal);
+		}
```
Indentation is not consistent

JasonStallkamp replied:
Other than that it is good to merge

JasonStallkamp reviewed (changes requested):
Fix indentation Local War/Assets/Resources/Script/CityGen/MeshGenerator.cs on line 191 to 208

tkwebster commented:
What's inconsistent in the indentation? The if statements? I just opened it up again and it looks correct to me.

JasonStallkamp commented on Local War/Assets/Resources/Script/CityGen/MeshGenerator.cs:
```diff
@@ -183,22 +188,87 @@ void BuildWall(float px, float py, float pz, float sx, float sy, float sz)
+		else 
+		{
+			cubeRenderer.material.SetColor("_Color", teal);
+		}
```
Here is the code with an extra indent

JasonStallkamp replied:
Could not find the indentation in the unity editor

tkwebster committed:
Indent fix

Rewrote this section in VS Code, still not sure what / why it was being flagged, hopefully this fixes it?

JasonStallkamp committed:
Merge branch 'dev' into CityGen2
